src/Viewer.tsx

```diff
@@ -17,58 +21,44 @@ export interface WidgetProps {
   width: number;
 }
 
-const agentColors = [
-  '#fee34d',
-  '#f7b232',
-  '#bf5736',
-  '#94a7fc',
-  '#ce8ec9',
-  '#58606c',
-  '#0ba345',
-  '#9267cb',
-  '#81dbe6',
-  '#bd7800',
-  '#bbbb99',
-  '#5b79f0',
-  '#89a500',
-  '#da8692',
-  '#418463',
-  '#9f516c',
-  '#00aabf',
-];
-
 function ViewerWidget(props: WidgetProps): JSX.Element {
+  const [modelInfo, setModelInfo] = useState<ModelInfo>({});
```

Comment:
oh, I see why you pulled that interface out from here. This seems like something that should exist in the viewer? Instead of having to redefine the interface here. I'll check

Reply:
It was missing, I added it in a few PRs. Since that will take a little bit of time, I would probably just type this as `interface modelInfo {[key]: string}` since it's all optional data anyway, with a TODO to use the TrajectoryFileInfo type exported from the viewer once it's been updated.